### PR TITLE
Import fenix quarantine

### PR DESCRIPTION
**Pull request by mozilla-pontoon**
Import fenix quarantine

### CODE REVIEW

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -2245,13 +2245,19 @@
...
     <!-- Caption text to be displayed in review quality check contextual onboarding card above the opt-in button. Parameter is the Fakespot product name. After the colon, what appears are two links, each on their own line. The first link is to a Privacy policy (review_quality_check_contextual_onboarding_privacy_policy_2). The second link is to Terms of use (review_quality_check_contextual_onboarding_terms_use_2). -->
     <string name="review_quality_check_contextual_onboarding_caption_2" moz:RemovedIn="123" tools:ignore="UnusedResources">By selecting “Yes, try it” you agree to the following from %1$s:</string>
+    <!-- Caption text to be displayed in review quality check contextual onboarding card above the opt-in button. First parameter is Firefox app name, third parameter is the Fakespot product name. Second & fourth are for clickable texts defined in review_quality_check_contextual_onboarding_privacy_policy_3 and review_quality_check_contextual_onboarding_terms_use. -->
+    <string name="review_quality_check_contextual_onboarding_caption_3" moz:RemovedIn="124" tools:ignore="UnusedResources">By selecting “Yes, try it” you agree to %1$s\'s %2$s and %3$s\'s %4$s.</string>
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml:2252]: @DreVla This string is still using the incorrect quote, so linter is complaining

**Review by Delphine — CHANGES_REQUESTED** [review]

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (DreVla) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml:2252]: @Delphine I landed a fix [here](https://github.com/mozilla-mobile/firefox-android/pull/5595), not sure what to do next. Should I just remove this completely?

**Comment by mozilla-pontoon** [comment]
Run ID: 7991270946
Authors:  @t-p-white

## TOML file: firefox.toml

### File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml

**ID**: `juno_onboarding_default_browser_title_nimbus_3`
**Text**: `Find out why millions love Firefox`
**Errors:**
- Hard-coded brand `Firefox`. Use a variable instead.

**Total errors not yet reported:** 1

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Delphine) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml:2256]: @DreVla `review_quality_check_contextual_onboarding_caption_4` looks good, but `review_quality_check_contextual_onboarding_caption_3` still has the wrong quotes there

**Review by Delphine — CHANGES_REQUESTED** [review]

**Comment by Delphine** [comment]
(btw I will add an exception to linter for the error on "Firefox")

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -2245,13 +2253,19 @@
...
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml:2260]:
```suggestion
    <string name="review_quality_check_contextual_onboarding_caption_3" moz:RemovedIn="124" tools:ignore="UnusedResources">By selecting “Yes, try it” you agree to %1$s\’s %2$s and %3$s\’s %4$s.
</string>
```

**Review by Delphine — CHANGES_REQUESTED** [review]

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml (hunk 1 above)
Comment (Delphine) [reply mozilla-mobile/fenix/app/src/main/res/values/strings.xml:2260]: My bad, we don't escape typographical apostrophes, so   `review_quality_check_contextual_onboarding_caption_4` also needs to be updated as suggested for `review_quality_check_contextual_onboarding_caption_3`

**Comment by mozilla-pontoon** [comment]
Run ID: 8098250345
Authors:  @sarah541

## TOML file: firefox.toml

### File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml

**ID**: `preferences_passwords_saved_logins_description_empty_text_2`
**Text**: `The passwords you save or sync to %s will be listed here. All passwords you save are encrypted.`
**Errors:**
- String was changed without a new ID. Previous value: `The passwords you save or sync to %s will be listed here. All passwords you save are encrypted.
`

**Total errors not yet reported:** 1

**Comment by sarah541** [comment]
> `preferences_passwords_saved_logins_description_empty_text_2`

I believe this error occurred because initially this string utilized 2 lines, but with removal of "UnusedResources" , it became a single line. There was no actual change to the string!
Will it be okay to ignore this one?

File: mozilla-mobile/fenix/app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -1621,47 +1635,46 @@
...
     <!-- Preference option for adding a login -->
-    <string name="preferences_logins_add_login">Add login</string>
+    <string name="preferences_logins_add_login" moz:RemovedIn="125" tools:ignore="UnusedResources">Add login</string>
     <!-- Preference option for adding a password -->
-    <string name="preferences_logins_add_login_2" tools:ignore="UnusedResources">Add password</string>
+    <string name="preferences_logins_add_login_2">Add password</string>
 
-    <!-- Preference for syncing saved logins in Fenix -->
-    <string name="preferences_passwords_sync_logins">Sync logins</string>
     <!-- Preference for syncing saved passwords in Fenix -->
```
Comment (Delphine) [inline mozilla-mobile/fenix/app/src/main/res/values/strings.xml:1642]: should be saved passwords

**Review by Delphine — APPROVED** [review]
Unblocking this PR from landing at this point. Will deal with nits accordingly, and later

**Pull request merged**